File: Server/Models/Message.cs
using System;
using System.Collections.Generic;
using Server.Context;

namespace Server.Models
{
    public class Message
    {
        public int Id { get; set; }

        public int ChatId { get; set; }

        public DateTime TimeStamp { get; set; }

        public string Content { get; set; }

        public User user { get; set; }

        public Message(){

        }


        public Message(DbModels.Message message){
            Id = message.Id;
            TimeStamp = message.TimeStamp;
            Content = message.Content;
            user = new User(message.ApplicationUser);
            ChatId = message.ChatId;
        }
    }
}

File: client/src/main/java/Business/Models/Message.java
package Business.Models;

import Acquaintence.IMessage;

import java.util.Date;

public class Message implements IMessage {

    private int ID;
    private int chatID;
    private String content;
    private Date timeStamp;
    private User user;


    public Message(int ID, String content, Date timeStamp, User user, int chatID) {
        this.ID = ID;
        this.content = content;
        this.timeStamp = timeStamp;
        this.user = user;
        this.chatID = chatID;
    }

    @Override
    public int getID() {
        return ID;
    }

    @Override
    public String getContent() {
        return content;
    }

    @Override
    public Date getTimeStamp() {
        return timeStamp;
    }

    @Override
    public User getUser() {
        return user;
    }

    public int getChatID() {
        return chatID;
    }
}
